routing.rst

```diff
@@ -1011,6 +1237,22 @@ the controllers of the routes:
 
 .. configuration-block::
 
+    .. code-block:: php-attributes
+
+        // src/Controller/BlogController.php
+        namespace App\Controller;
+
+        use Symfony\Component\Routing\Annotation\Route;
+
+        class BlogController
+        {
+            #[Route('/blog/{page}', name: 'blog_index', defaults: ['page' => 1, 'title' => 'Hello world!'])]
```

Comment:
💡 I've kept this attribute single-line in order to remain consistent with the existing annotation block. But I think, the example would be more readable with we defined the attribute/annotation multi-line.

Reply:
I prefer multiline too, please adjust the examples. If you like here or in a separate PR first, your choice 😃